apps/vscode/src/lsp/client.ts

```diff
@@ -280,64 +272,61 @@ function embeddedGoToDefinitionProvider(engine: MarkdownEngine) {
   ): Promise<Definition | LocationLink[] | null | undefined> => {
     const vdoc = await virtualDoc(document, position, engine);
     if (vdoc) {
-      const vdocUri = await virtualDocUri(vdoc, document.uri, "definition");
```

Comment:
This diff looks ugly but I didn't change anything in the `try` or `catch` block. It's just switching to `withVirtualDocUri()` and removing the `finally` block